jetstream/engine/token_utils.py

```diff
@@ -349,6 +349,11 @@ def bos_id(self) -> int:
     """ID of the BOS token."""
     return self.vocab.bos_id
 
+  @property
```

Comment:
Unless we want to return different stop_tokens, we don't need to this function in child class. 

Remove the stop_tokens function here does not affect behavior. By default, it will call parent  stop_tokens.

Reply:
we want to return different stop_tokens for TikToken: https://github.com/google/JetStream/blob/main/jetstream/third_party/llama3/llama3_tokenizer.py#L94

Reply:
done.